### ISSUE TITLE
FR: Support Multi-episode files

### ISSUE THREAD

**Issue by karan** [jasongdove/ErsatzTV#1164]
Currently I see errors like "Multi-episode files are not yet supported" during library scan. Creating this to track adding support for multi-episode files.

**Comment by jasongdove** [comment]
Some additional context: this used to work before paging was added to media server scans; ETV would merge each episode's metadata entries into one "episode" in ETV's database. Different logic is now required since there are no guarantees that a single page will return all episodes within a multi-episode file.

**Comment by karan** [comment]
Interesting good to know. Would local media work with multi episode files? Is that better than Plex integration?

**Comment by jasongdove** [comment]
Yes, local libraries should properly identify multi-episode files and include metadata for both episodes in ETV's UI. That's really the only limitation of the Plex integration, other than the annoying SQL errors: when browsing the season in ETV's UI, you will only see one of the two items.

With local libraries or with Plex (or Jellyfin or Emby), multi-episode files will always be played in their entirety. So, unless you're bothered by only seeing one of two episodes in the season view in ETV, there isn't really any functionality missing.

**Comment by karan** [comment]
Oh I see. So in any source, the UI is what's glitchy but the playback isn't. 

Okay so then this is less of a priority IMO

**Comment by jasongdove** [comment]
Somewhat related, this can cause problems if you switch from individual files to multi-episode files in a media server library.

from

```
S01E01 => /media/Show/Season 1/Show - s01e01.mkv
S01E02 => /media/Show/Season 1/Show - s01e02.mkv
```

to

```
S01E01 => /media/Show/Season 1/Show - s01e01-02.mkv
S01E02 => /media/Show/Season 1/Show - s01e01-02.mkv
```

The unique constraint causes ETV to remember the old path for one item which gives an error when playback is attempted, and will need to be fixed. I think I can address this as part of larger support for multi-episode files from media servers.

**Comment by jasongdove** [comment]
I am cleaning up GitHub issues and migrating feature requests to the ErsatzTV Fider instance.

This has moved to https://features.ersatztv.org/posts/9/support-multi-episode-files-from-media-servers
